subprojects/code-quality/src/integTest/groovy/org/gradle/api/plugins/quality/checkstyle/CheckstylePluginIntegrationTest.groovy

```diff
@@ -33,20 +33,20 @@ class CheckstylePluginIntegrationTest extends WellBehavedPluginTest {
      */
```

Comment:
You would want to remove the blurb above, right?

Reply:
I missed this.  Yes, I will remove.